content/basics/ably.textile

```diff
@@ -0,0 +1,86 @@
+---
+title: What is Ably?
+meta_description: "Ably is a realtime platform delivering powerful digital realtime experiences through a publish/subscribe network."
+---
+
+Ably’s realtime platform provides a scalable, fault-tolerant and worldwide publish/subscribe(pub/sub) network. Providing SDKs in a variety of programming languages with both REST and realtime interfaces Ably helps you get connected.
+
+This section will provide a short introduction to some of the key features and APIs of Ably, giving you an idea of what you can expect to build with Ably and how Ably's infrastructure helps you succeed.
+
+h2(#what). What can you build with Ably?
+
+Ably enables the distribution of realtime data to multiple users using the pub/sub messaging pattern with WebSockets, enabling you to build realtime experiences for your applications without having to build your own realtime infrastructure. Common use cares are:
+
+* Live Chat - chat for customer support or community engagement
+* Data Broadcast - broadcast realtime event data to millions of devices
+* Data Sync - keep backend and frontend clients in realtime sync
+* Multiplayer Collaboration - collaborative workspaces for synchronous teamwork
+* Notifications - deliver in-app notifications in realtime to users
+
+h2(#about). Ably features
+
+Ably provides a number of features that enhance realtime experiences, some of the key Ably features are: 
+
+* "Pub/sub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push Notifications":#push-notifications
+
+h3(#pubsub). Pub/sub with channels
+
+Ably uses the pub/sub architectural pattern to enable any number of publishers to publish data and any number of subscribers to subscribe to that data. This works with Ably's "concept of channels":/channels, enabling publishers to publish to a single location and subscribers to subscribe to a single source.
```

Comment:
Would it be good to give an example here? something like "pub/sub could be used to power live data feeds such as sports scores, travel updates, stocks, currency or auction prices."